.env.development

```diff
@@ -7,7 +7,7 @@
 DATABASE_URL=postgresql://postgres@localhost/laa_assess_non_standard_magistrate_fee_dev
 
 # Local datastore url - default is http://localhost:8000
-APP_STORE_URL=http://localhost:8000
+APP_STORE_URL=https://localhost:3002
```

Comment:
can we remove this for the same reasons as per similar PR. your local preferences can be put in `env.development.local` if you need to or want to try stuff out